src/dependency-manager/src/component-config/v1.ts

```diff
@@ -25,16 +25,40 @@ export type ParameterValueSpecV1 = string | number | boolean | ParameterDefiniti
 
 interface ServiceContextV1 {
   environment: Dictionary<string>;
-  interfaces: Dictionary<ServiceInterfaceSpec>;
+  interfaces: Dictionary<InterfaceSpec>;
 }
 
 export interface ComponentContextV1 {
   dependencies: Dictionary<ComponentContextV1>;
   parameters: Dictionary<ParameterValue>;
-  interfaces: Dictionary<ServiceInterfaceSpec>;
+  interfaces: Dictionary<InterfaceSpec>;
   services: Dictionary<ServiceContextV1>;
 }
 
+export function transformServices(input?: Dictionary<object | ServiceConfigV1>): Dictionary<ServiceConfigV1> {
```

Comment:
So it looks like we just moved this here from `service-config/v1.ts` because it's on the the getter now which is here, correct?

I'm increasingly a fan of this pattern.

Reply:
We used to use to transform dependencies in service-config, but now it is only used in component-config.